### PR TITLE
CORE-7908: Always Capture Test Result

### PR DESCRIPTION
**Pull request by knguyen-r3**
* Added an always statement into 'smoketests' stage and moved the capturing of Junit results into this stage.

### CODE REVIEW

File: .ci/e2eTests/JenkinsfileCombinedWorker [hunk 1]
```diff
@@ -134,11 +134,15 @@ pipeline {
...
-               gradlew('smoketest -PisCombinedWorker=true')
+            always {
+                options {
+                    timeout(time: 15, unit: 'MINUTES')
+                }
+                steps {
+                   gradlew('smoketest -PisCombinedWorker=true')
+                }
+
+                junit allowEmptyResults: true, testResults: '**/test-results/**/TEST-*.xml'
```
Comment (ronanbrowne) [inline .ci/e2eTests/JenkinsfileCombinedWorker]: needs to be in a post stage see example [here ](https://github.com/corda/corda-runtime-os/blob/release/os/5.0/.ci/e2eTests/Jenkinsfile#L125-L134)

**Review by ronanbrowne — CHANGES_REQUESTED** [review]
see comments

**Review by ronanbrowne — APPROVED** [review]
LGTM

**Review by yarivus — COMMENTED** [review]
thx @ronanbrowne

**Pull request merged**